.github/workflows/build-publish-pypi.yml

```diff
@@ -0,0 +1,77 @@
+name: Build KeplerGL Python and NPM Packages
+
+on: push
```

Comment:
should this be `on:push` or `on:release`?

Reply:
I think it should be “on:push”, which means this action can be used to build, check and test keplergl python package on push with changes. For release and publish, there is a step: “-name: Check Release Tag”.

Reply:
Also, see the updated steps in “RELEASE.md” : https://github.com/lixun910/kepler.gl/blob/build-pypi-conda/bindings/kepler.gl-jupyter/RELEASE.md